etcd-manager/pkg/etcd/restore.go

```diff
@@ -143,17 +143,37 @@ func RunEtcdFromBackup(backupStore backup.Store, backupName string, basedir stri
 		return nil, err
 	}
 
-	// TODO: randomize port
-	port := 8002
-	peerPort := 8003 // Needed because otherwise etcd won't start (sadly)
-	clientUrl := "https://127.0.0.1:" + strconv.Itoa(port)
-	peerUrl := "https://127.0.0.1:" + strconv.Itoa(peerPort)
 	myNodeName := "restore"
 	myNode := &protoetcd.EtcdNode{
-		Name:       myNodeName,
-		ClientUrls: []string{clientUrl},
-		PeerUrls:   []string{peerUrl},
+		Name: myNodeName,
 	}
+
+	currentDir := ""
+
+	if !isV2 {
```

Comment:
I thought I removed all support for V2. Should we remove this part also?

Reply:
Good point - removed all V2 support from this function - it wouldn't work anyway as we don't ship etcd2!

Reply:
:)